content/_index.md

```diff
@@ -39,3 +39,23 @@ There is an additional option to purchase a Planet Nix swag bag, but this is not
 Wondering what to expect at Planet Nix? Watch the talks from [NixCon NA 2024](https://2024-na.nixcon.org/recordings/).
 
 For any questions, please contact us at [info@planetnix.com](mailto:info@planetnix.com).
+
+## **FAQ**
+
+### GitHub API
+
+With the number of people at the conference, it is easy to reach the rate limit of the GitHub API. This will affect everyone at the conference. To help mitigate this, it is recommended to add a GitHub API Token to your NixOS configuration. Below are two ways this can be accomplished.
```

Comment:
It would be a good idea to call out what kind of token. Since github just wants to verify the user you actually dont need any extra permissions (which is good). Nixcon 2024 called this out pretty succinctly: https://2024.nixcon.org/faq/ in there: `Do I need to prepare anything?`

I think itd also be a good idea to link to the nix docs for acces-token like they do on their FAQ too: https://nix.dev/manual/nix/2.24/command-ref/conf-file.html#conf-access-tokens